node/cn/snap/testrand/rand.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package testrand
```

Comment:
Let's move this file under `common/testrand/rand.go`. In geth it's located in `internal/testrand/rand.go` and used throughout various packages. Since we don't have `internal`, I think `common` is second best place to put this utility.

Reply:
Moved. https://github.com/kaiachain/kaia/pull/125/commits/b1f2842e94baeaf3963d7e1ec41db1493be30985